static/locales/en-US.yaml

```diff
@@ -541,6 +541,15 @@ Video:
       reversing playlists: reversing playlists
       shuffling playlists: shuffling playlists
       looping playlists: looping playlists
+  Stats:
+    video id: "Video ID (youtube)"
+    player resolution: "Viewport"
+    volume: "Volume"
+    fps: "Fps"
```

Comment:
FPS

Reply:
@constraintAutomaton You did not change this in last commit